app/controllers/hyrax/admin/permission_template_accesses_controller.rb

```diff
@@ -31,6 +26,30 @@ def source_id
         def update_management
           Forms::PermissionTemplateForm.new(@permission_template_access.permission_template).update_management
         end
+
+        def redirect_to_edit_path
+          if Hyrax::PermissionTemplate.find(@permission_template_access.permission_template_id).source_type == 'admin_set'
```

Comment:
The following query is duplicated: 
`Hyrax::PermissionTemplate.find(@permission_template_access.permission_template_id).source_type`

Please extract a common method for this.

Reply:
I've made the requested changes and the PR is green again